models/unit.js

```diff
@@ -66,9 +66,9 @@ class Unit {
             subject_id AS "subjectId",
             number,
             title,
-            start_date AS "startDate",
-            end_date AS "endDate",
-            review_date AS "reviewDate",
+            to_char(start_date, 'YYYY-MM-DD') AS "startDate",
```

Comment:
isn't this the default format for postgres date types?

Reply:
without formatting, the output contains the time as well, e.g. 2021-04-24T00:00:00.000Z, and only the year-month-day is needed on the frontend